### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by Ranjeetk489**

### CODE REVIEW

File: components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  margin: 10px;
+  padding: 1rem;
+  border-radius: var(--border-radius);
+  color: var(--white-color);
+  cursor: pointer;
+  background-color: white;
+  width: 85%;
+}
+.alert-icon {
+  margin-right: 0.5rem;
```
Comment (developer-anand) [inline components/Alert/alert.css]: use px for values less than 10px

File: css/sidebar.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  background-color: var(--complementary-color);
+  color: var(--tertiary-color);
+  margin-bottom: 1rem;
+  cursor: pointer;
+  border-radius: 5px;
+}
+.sidebar-item-btn:hover {
+  box-shadow: 0 0 1.2rem #719ece;
+}
+
```
Comment (developer-anand) [inline css/sidebar.css:24]: remove extra spaces, just be consistent with spacing it would help in code structure.

**Review by developer-anand — APPROVED** [review]
Good use of CSS variables Can be merged with suggested changes.

File: components/Alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
```
Comment (sudipkundu999) [inline components/Alert/alert.html]: change title

**Review by sudipkundu999 — APPROVED** [review]
Good use of css variables.
Good to merge!

**Pull request merged**